docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -309,6 +318,20 @@ repository as well as each subsequent push to each pull request.
 To pass information from the upstream patch to the downstream patch use
 [downstream_expansions.set](Project-Commands.md#downstream_expansionsset)
 
+**Example**:  to allow testing Spruce tasks as part of patches for the Evergreen project,  
+a configuration like this could be added to **Evergreen's project page:**
+
+![patch_trigger_alias_example.png](../images/patch-trigger-alias-example.png)
+
+This makes it possible to optionally add tasks matching the defined regex to any patch, and because 
```

Comment:
"is checked, meaning that these tasks will be included " 
--> 
"is checked these tasks will be included "